0-bootstrap/sa.tf

```diff
@@ -111,6 +111,7 @@ locals {
     ],
     "proj" = [
       "roles/storage.objectAdmin",
+      "roles/storage.admin",
```

Comment:
Is this the least privilege possible?

Reply:
Yes, It's necessary to have storage.admin role, otherwise we'll get permission denied when we are building the docker confidential image in the 4-projects.

Reply:
Shouldn't the docker image get built inside a project created for step 5, inside 4-projects to limit the need to add this very permissive role?